heron/packing/src/java/org/apache/heron/packing/binpacking/TwoDimensionVectorBinPacking.java

```diff
@@ -0,0 +1,379 @@
+/**
```

Comment:
I am feeling that it is fine to keep the old name, since it is BinPacking anyway. Also if any user was using BinPacking before, renaming would require them to update their config which is not convenient. The new code works very similar to the old logic anyway.

Just my 2 cents.

Reply:
That actually sounds perfect to me. I prefer not to have too many packing classes too.